src/pages/Ecoverse.tsx

```diff
@@ -51,9 +50,16 @@ const useStyles = createStyles(theme => ({
 
 interface EcoversePageProps extends PageProps {
   ecoverse: EcoverseInfoQuery;
+  permissions: {
```

Comment:
If all the pages use the `permissions` field, it's better to have it in `PageProps` or extend it from a different interface.

Reply:
Can it be defined as `permissions: Record<string, boolean>`?

Reply:
defining it as Record<string, boolean> makes it not specific enough. Better global `Authorization` functionality is required.